src/Evolution/Systems/RadiationTransport/M1Grey/Sources.hpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+
+namespace RadiationTransport {
+namespace M1Grey {
+
+/*!
+ * \brief Compute the curvature source terms for the flux-conservative
+ * grey M1 radiation transport.
+ *
+ *
+ * A flux-conservative system has the generic form:
+ * \f[
+ * \partial_t U_i + \partial_m F^m(U_i) = S(U_i)
+ * \f]
+ *
+ * where \f$F^a()\f$ denotes the flux of a conserved variable \f$U_i\f$ and
+ * \f$S()\f$ denotes the source term for the conserved variable.
+ *
+ * For the grey M1 formalism (neglecting coupling to the fluid):
+ * \f{align*}
+ * S({\tilde E}) = & \alpha \tilde P^{ij} K_{ij} - \tilde F^i \partial_i \alpha,
+ * S({\tilde F_i}) = & -\tilde E \partial_i \alpha + \tilde F_k \partial_i
+ * \beta^k
+ * + \frac{1}{2} \lapse \tilde P^{ij} \partial_i \gamma_{jk},
+ * \f}
+ *
+ * where \f${\tilde E}\f$, \f${\tilde F_i}\f$, \f${\tilde P}^{ij}\f$ are the
```

Comment:
Can we use a different symbol than F for the densitized momentum (we are already using it for the flux) perhaps S for consistency with the fluid variables

Reply:
I'm ok either way. Using tildeF is consistent with the M1 literature, but on the other hand tildeF and tildeS are the exact same projection of the stress-energy tensor (one for the MHD part, the other for the radiation part), so there is certainly some value to using the same notation. 
If we do that, we may want to also use uniform notation for the pressure tensor  (S^{ij} in the fluid notation, P^{ij} in the neutrino notation), for consistency.